tcp_sink.go

```diff
@@ -115,16 +108,61 @@ func (s *tcpStatsdSink) handleFlushError(err error, droppedBytes int) {
 	})
 }
 
+func (s *tcpStatsdSink) flushUint64(name, suffix string, u uint64) {
+	b := pbFree.Get().(*buffer)
+
+	b.WriteString(name)
+	b.WriteChar(':')
+	b.WriteUnit64(u)
+	b.WriteString(suffix)
+
+	s.mu.Lock()
+	if _, err := s.bufWriter.Write(*b); err != nil {
+		s.handleFlushError(err, s.bufWriter.Buffered())
+	}
+	s.mu.Unlock()
+
+	*b = (*b)[:0]
+	pbFree.Put(b)
+}
+
+func (s *tcpStatsdSink) flushFloat64(name, suffix string, f float64) {
```

Comment:
private methods beneath public methods?

Reply:
I think it makes sense here because the implementation is nearly the same as `flushUint64()` (so it's nice having them side-by-side) and `FlushTimer()` calls either `flushUint64()` or `flushFloat64()` so this makes the already slightly tricky `FlushTimer()` logic simpler.  

Also, I think this is fine in general.

Reply:
i was merely referring to the placement in the file.